### ISSUE TITLE
Training issue

### ISSUE THREAD

**Issue by rangerli** [argman/EAST#69]
hi, when i use EAST to detect other objects such as ship, i find so many problems. When training my own data, which python version should i use(2 o r 3).
![img_1](https://user-images.githubusercontent.com/23238816/33062368-031f61c6-ceda-11e7-91f5-4c6709cc8912.jpg),
![image](https://user-images.githubusercontent.com/23238816/33062383-12c64626-ceda-11e7-9a37-a4beec5ff352.png).here are my examples.

**Comment by argman** [comment]
whats your problems?

**Comment by rangerli** [comment]
Firs, I used Python 3.5 and installed the package followed the 'requirements,txt'. Now, when I started the multigpu_train.py , I met these problems.
![image](https://user-images.githubusercontent.com/23238816/33073244-3f580724-cefd-11e7-9bfe-32d32478d75e.png)
![image](https://user-images.githubusercontent.com/23238816/33073250-42d291da-cefd-11e7-8ac6-1ff4c1fb1305.png)
![image](https://user-images.githubusercontent.com/23238816/33073254-46ec4086-cefd-11e7-8c7f-8617d0d6aba4.png)

**Comment by argman** [comment]
the warning can be ignored, you can check the code when reading your annotation file.

**Comment by rangerli** [comment]
Thanks for reply. Now I have run the train.py. But the loss doesn't change.
![image](https://user-images.githubusercontent.com/23238816/33114725-189cf260-cf99-11e7-8c87-b138bde73b87.png)

**Comment by argman** [comment]
can you paste the visualization of tensorboard ?

**Comment by anushaGundapaneni** [comment]
when i'am trying to run **multigpu_train.py** , i'am just trying with **training samples** images and training is not yet started.From last 3 days this is the progress . I dont know why it is not starting . 

I dont know how **your training** started . Please can you suggest ?
![screenshot from 2017-11-27 10-08-25](https://user-images.githubusercontent.com/31063969/33251189-66f84926-d35b-11e7-975d-1f171d6d6cc9.png)

**Comment by anushaGundapaneni** [comment]
I have following queries. Please do respond and provide some suggestions ::

1.should the image(jpg) and as well as (.txt) file should be there in the same folder or in different folders?
2. how much time it takes to complete 1 step ?
3. How many steps required to obtain better accuracy ? 

Can you please help me with these questions !

**Comment by yingning** [comment]
@anushaGundapaneni ,hello friend ,I have the same problem ,have you fixed this problem?

**Comment by anushaGundapaneni** [comment]
@yingning    

Yes,  
The problem is Dataset is not loading !

In order to load that you need to change the file name of (gt_img_*.txt) to (img_*.txt) and text files and images should be there in the same folder .

**Comment by yingning** [comment]
I have changed the file name of (gt_img_.txt) to (img_.txt) and put the text files and images in the same folder ,but I still have  the problem ,and my computer was stuck when i run the multigpu_train.py .Should I change some code in the project?Any help will be appreciated! @anushaGundapaneni

**Comment by anushaGundapaneni** [comment]
no need to change code. when you start training **"multigpu_train.py"** it takes some  time to start training steps after loading images i.e., 
2 training images loaded in /path/****
2 training images loaded in /path/****
2 training images loaded in /path/****
***
**

After this you need to wait untill **steps** starts.

**Comment by yingning** [comment]
First , thank you for your reply.How long it will take until the steps starts when I start run "multigpu_train.py"? And I found my computer was stuck a long time  after  I run  "multigpu_train.py" .After  at least 20 minutes I  found  I can do nothing on it but  reboot the computer. @anushaGundapaneni

**Comment by gangliu123** [comment]
@yingning

**Comment by gangliu123** [comment]
@yingning   Hi, have you solve that one yet? I got the same issue

**Comment by KienVu2368** [comment]
I think it because number of training image. If you increase your training image up to 20, script will run as normal.

**Comment by xiaodao1990** [comment]
@KienVu2368 when 1 increase my image up to 20, script can run. tks.
